test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/047_ATF_P_TC_navi_PTU_brings_certificate_RPC_ForceProtectedService_ON.lua

```diff
@@ -0,0 +1,168 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] PolicyTableUpdate is valid and brings "certificate"
+-- [PTU] [GENIVI] SDL must start PTU for navi app right after app successfully registration
+--
+-- Description:
+-- In case SDL starts PolicyTableUpdate in case of no "certificate" at "module_config" section at LocalPT
+-- and PolicyTableUpdate is valid and brings "certificate" at "module_config" section
+-- and app sends StartService (<any_serviceType>, encypted=true) to SDL
+-- SDL must start TLSHandshake sequence (by sending TLS Hello to this mobile app)
+--
+-- 1. Used preconditions:
+-- ForceProtectedService is set to ON(0x07, 0x0A, 0x0B, 0x0F) in .ini file
+-- Navi app exists in LP, no certificate in module_config
+-- Register and activate navi application.
+-- -> SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- Send correct policy file, certificate exists in module_config
+-- -> SDL sends SDL.OnStatusUpdate(UP_TO_DATE)
+--
+-- 2. Performed steps
+-- Send RPC service.
+--
+-- Expected result:
+-- SDL should return StartServiceACK, encrypt = true to RPC
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"NAVIGATION"}
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local mobile_session = require('mobile_session')
+local testCasesForPolicyCeritificates = require('user_modules/shared_testcases/testCasesForPolicyCeritificates')
+local events = require('events')
+local Event = events.Event
+
+--[[ General Precondition before ATF start ]]
+commonPreconditions:BackupFile("smartDeviceLink.ini")
+commonFunctions:write_parameter_to_smart_device_link_ini("ForceProtectedService", "0x07, 0x0A, 0x0B, 0x0F")
+testCasesForPolicyCeritificates.update_preloaded_pt(config.application1.registerAppInterfaceParams.appID, false)
+testCasesForPolicyCeritificates.create_ptu_certificate_exist(true, false)
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_resumption')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_connectMobile()
+  self:connectMobile()
+end
+
+function Test:Precondition_StartSession()
+  self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession:StartService(7)
```

Comment:
@istoimenova, Shouldn't we use _testCasesForPolicyCeritificates.StartService_encryption(self,7)_ here?